assimilate.csh

```diff
@@ -637,6 +645,8 @@ if ($USING_PRIOR_INFLATION == true) then
    else
       # Look for the output from the previous assimilation (or fill_inflation_restart)
       # If inflation files exists, use them as input for this assimilation
+      # TODO: This should probably look for a date, instead of relying on the files
+      #       we want being the youngest in the directory.
```

Comment:
So - again ... make sure the extra inflation files are moved out of the way ...

Reply:
Good catch.  The inflation files are ready.